### DISCUSSION TITLE
Gente não tô conseguindo mudar a senha e não lembro dela pra mudar precisa saber a senha antiga e eu não lembro

### DISCUSSION THREAD

**Discussion by Monikoliveiracorrea** [iotaledger/firefly#7049] [Help & Support]
Por favor gentee ajuda não estou conseguindo mudar senha pois não lembro da outra

**Comment by begonaalvarezd** [comment]
Hi, if you forgot your password you can create a new profile using the same mnemonic/seed and set a new password & pin for the new profile.
Hope this helps 🙏🏼
